docs/examples/nlohmann_json_serialize_enum_3.cpp

```diff
@@ -0,0 +1,47 @@
+#include <iostream>
+#include <nlohmann/json.hpp>
+
+using json = nlohmann::json;
+
+namespace ns
+{
+enum class Color
+{
+    red, green, blue, yellow
+};
+
+NLOHMANN_JSON_SERIALIZE_ENUM_STRICT(Color,
+{
+    { Color::green, "green" },
+    { Color::blue, "blue" },
+    { Color::red, "rot" } // note that serialization for yellow is missing
```

Comment:
Should this say `"red"` instead of `"rot"`?

Reply:
yes, changed